src/app/components/pages/Glossary.tsx

```diff
@@ -206,24 +277,45 @@ export const Glossary = () => {
             <Row>
                 <Col md={{size: 9}} className="py-4">
                     <Row className="no-print">
+                        {isPhy && <Col className="mt-3 mt-md-0">
+                            <Label for='subject-select' className='sr-only'>Subject</Label>
+                            <StyledSelect inputId="subject-select"
+                                options={ subjects.map(e => ({ value: e.id, label: e.title})) }
+                                value={querySubjects ? ({value: querySubjects.id, label: querySubjects.title }) : undefined}
+                                name="subject-select"
+                                placeholder="Select a subject"
+                                onChange={e => setFilterSubject(subjects.find(v => v.id === (e as Item<TAG_ID> | undefined)?.value)) }
+                                isClearable
+                            />
+                        </Col>}
                         <Col md={{size: 4}}>
                             <Label for='terms-search' className='sr-only'>Search by term</Label>
                             <Input
                                 id="terms-search" type="search" name="query" placeholder="Search by term" aria-label="Search by term"
                                 value={searchText} onChange={e => setSearchText(e.target.value)}
                             />
                         </Col>
-                        <Col className="mt-3 mt-md-0">
+                        {isAda && <Col className="mt-3 mt-md-0">
                             <Label for='topic-select' className='sr-only'>Topic</Label>
                             <StyledSelect inputId="topic-select"
                                 options={ topics.map(e => ({ value: e.id, label: e.title})) }
                                 name="topic-select"
-                                classNamePrefix="select"
                                 placeholder="All topics"
                                 onChange={e => setFilterTopic(topics.find(v => v.id === (e as Item<TAG_ID> | undefined)?.value)) }
                                 isClearable
                             />
-                        </Col>
+                        </Col>}
+                        {isPhy && <Col className="mt-3 mt-md-0">
+                            <Label for='stage-select' className='sr-only'>Stage</Label>
+                            <StyledSelect inputId="stage-select"
+                                options={ stages.map(e => ({ value: e, label: stageLabelMap[e]})) }
```

Comment:
I would just try to keep the naming of the anonymous parameters consistent. The convention is the singular of the array name / first letter of, `e` in particular is mostly reserved for events so looks a bit weird here.
```suggestion
                                options={ stages.map(s => ({ value: s, label: stageLabelMap[s]})) }
```